README.md

```diff
@@ -34,29 +34,30 @@
   <ul>
     <li>HTML</li>
     <li>CSS</li>
+    <li>JavaScript</li>
   </ul>
 </details>
 
 ### Key Features <a name="key-features"></a>
 
-- **Added a navbar**
-- **Added a style for home page using flexbox**
-- **Added a background image for my homepage**
-- **Added some bio about me**
+- **Added home page**
+- **List of known donators**
+- **Added about page**
+- **Added account status show on navbar**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://ubdanam.github.io/Portfolio/)
+- [Live Demo Link](https://ubdanam.github.io/Fundraising-platform/)
```

Comment:
- [OPTIONAL] KIndly make sure to deploy again your project because the link is broken. Kindly make sure to fix it in order to keep your repo professional.